code/modules/robotics/robot/module_tool_creator/modules.dm

```diff
@@ -55,16 +56,18 @@
 		/obj/item/device/igniter,
 		/obj/item/saw/cyborg,
 		/obj/item/satchel/hydro, // TODO: make more versatile version
+		/obj/item/paper_bin/robot,
```

Comment:
Would probably argue un-needed as paper is damned-well everywhere, and can even be grown on trees in botany. I'm fine with its inclusion for now, can always mess around with later.